README.md

```diff
@@ -70,25 +70,23 @@ We use [build](https://pypa-build.readthedocs.io/en/latest/) toolchain to build
 If we have to update the tool, use the following steps
 
 ```shell
-# First create a new fresh virtualenv
-rm -rf .venv && python3 -m venv .venv
+# Ensure you are running in a cleanly boostrapped virtual environment
+rm -rf .venv
+make install-deps
 source .venv/bin/activate
-# Then install pip-tools, from pinned dependencies
-python3 -m pip install -r workstation-bootstrap/requirements.txt
-# Then update the requirements.in file as required
-pip-compile --allow-unsafe --generate-hashes \
-    --output-file=workstation-bootstrap/requirements.txt workstation-bootstrap/requirements.in
-# Now we are ready for bootstrapping
-./scripts/build-sync-wheels --project workstation-bootstrap --pkg-dir ./workstation-bootstrap --requirements .
-# Here we have the new wheels ready
-# Now let us recreate our new sha256sums for bootstrapping
+# Update the workstation-bootstrap/pyproject.toml file as required.
+# Then update the lockfile:
+poetry -C workstation-bootstrap/ lock
```

Comment:
I think the workflow should be either: run `poetry update <name>` OR update pyproject.toml + `poetry lock --no-update`

Reply:
Makes sense to me, fleshed out README examples a bit.